pkg/internal/computeresources/transformer_test.go

```diff
@@ -388,6 +388,112 @@ func TestTransformerOneContainer(t *testing.T) {
 				},
 			}},
 		},
+	}, {
+		description: "limitRange with min and max, while resources on containers < min",
```

Comment:
Was this a bug? Could you make this change in a separate PR?

Reply:
sure. and saw your raised refact PR.

think good for me to keep "updating pod" logic in this PR while leaving the transformer and limitrange changes.

Reply:
a follow-up PR I will apply on "setting the requests and limits" after this refact PR was merged.

Reply:
raised here - https://github.com/tektoncd/pipeline/pull/5269